cp2k_output_tools/blocks/energies.py

```diff
@@ -13,9 +30,16 @@
 
 
 def match_energies(content: str) -> Optional[BlockMatch]:
+    spans = []
+    energies = {}
+    match = MAIN_ENERGY_RE.search(content)
+    if match:
+        energies = match.groupdict()
+        spans = match.spans(0)
     match = FORCE_EVAL_ENERGY_RE.search(content)
-
-    if not match:
+    if match:
+        energies["total force_eval"] = match["value"]
+        spans += match.spans(0)
+    if len(energies) == 0:
```

Comment:
```suggestion
    if not energies:
```
Empty `dict` evaluates to `False`.